server/remote/msgraph/get_me.go

```diff
@@ -4,15 +4,13 @@
 package msgraph
 
 import (
-	"strings"
-
 	"github.com/mattermost/mattermost-plugin-mscalendar/server/serializer"
 
 	"github.com/pkg/errors"
 )
 
 const (
-	ErrorUserInactive        = "You have been marked inactive. Please disconnect and reconnect your account again."
+	ErrorUserInactive        = "You have been marked inactive because your refresh token is expired. Please disconnect and reconnect your account again."
```

Comment:
This actually reads kind of weird, if this is an error we show to the user. I think we can leave that part out of this string. We can log a different string for this. Also, I think we should move these strings to a different file since they're not used in this file.